bootstrap/bootstrap.ps1

```diff
@@ -45,6 +45,11 @@ Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/script
 $rootCliVersion = Join-Path $RepositoryRoot ".cliversion"
 $dotNetCliVersion = Get-Content $rootCliVersion
 
+if (-Not (Test-Path $CliLocalPath))
```

Comment:
Probably- I'll follow up. This won't pass CI without the fix as it pulls directly from the repo. I assume I can put this in?